draft-ietf-quic-manageability.md

```diff
@@ -299,38 +301,36 @@ packet for QUIC to be used.
 The content of Initial packets is encrypted using Initial Secrets, which
 are derived from a per-version constant and the client's destination connection
 ID; they are therefore observable by any on-path device that knows the
-per-version constant. They are therefore considered visible in this
-illustration. The content of QUIC Handshake packets are encrypted using keys
-established during the initial handshake exchange, and are therefore not
-visible.
+per-version constant and considered visible in this illustration. The content
+of QUIC Handshake packets are encrypted using keys established during the
+initial handshake exchange, and are therefore not visible.
 
-Initial, Handshake, and the 1-RTT packets transmitted after the handshake
-belong to cryptographic and transport contexts. The Client Completion
-{{fig-init-complete}} and the Server Completion {{fig-hs-complete}} datagrams
-finish these first two contexts, by sending the final acknowledgment and
-finishing the transmission of CRYPTO frames.
+Initial, Handshake, and 1-RTT packets belong to cryptographic and transport
+contexts. The Client Completion {{fig-init-complete}} and the
+Server Completion {{fig-hs-complete}} flights conclude these first two
+contexts, by sending final acknowledgments and CRYPTO frames.
```

Comment:
```suggestion
Server Completion {{fig-hs-complete}} flights conclude the Initial and Handshake
contexts, by sending final acknowledgments and CRYPTO frames.
```

If you look at this, it's not a great match to reality.  The Initial context concludes at the server when it sends its first Handshake packet.  Is this "lies to children" and therefore acceptable?

Reply:
Yeah, this was my best effort at rewriting what was there to be clearer and more accurate, but it's still a bit misleading or possibly just not helpful.

I'm not sure if the intent is to detail when no more packets from an encryption level will be sent, because the keys have been discarded?